internal/pkg/levee/levee.go

```diff
@@ -70,18 +69,21 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
-func reportSourcesReachingSink(pass *analysis.Pass, propagations map[*source.Source]propagation.Propagation, sink ssa.Instruction) {
-	for source, prop := range propagations {
+func reportSourcesReachingSink(conf *config.Config, pass *analysis.Pass, propagations map[*source.Source]propagation.Propagation, sink ssa.Instruction) {
+	for src, prop := range propagations {
 		if prop.IsTainted(sink) {
-			report(pass, source, sink.(ssa.Node))
+			report(conf, pass, src, sink.(ssa.Node))
 			break
 		}
 	}
 }
 
-func report(pass *analysis.Pass, source *source.Source, sink ssa.Node) {
+func report(conf *config.Config, pass *analysis.Pass, source *source.Source, sink ssa.Node) {
 	var b strings.Builder
 	b.WriteString("a source has reached a sink")
```

Comment:
why cant this be part of the format string below?

Reply:
This following is pretty readable to me:-
fmt.Fprintf(&b, "a source has reached a sink \n source: %v", pass.Fset.Position(source.Pos()))
but I don't feel too strongly about this, so its ok to keep it like this.

Reply:
\shrug.

I tend to go towards using additional lines when we have `\n` in the format, just to visualize the `\n`.

I suppose if we wanted to go in the entire other direction, we could collapse the entire method to

```golang
msg :=  strings.Trim(fmt.Sprintf("a source has reached a sink\n source: %v\n %v", pass.Fset.Position(source.Pos()), conf.ReportMessage), "\n "))
pass.Reportf(sink.Pos(), msg)
```

I was really just doing a minimal diff and keeping the style that was already there with the string buffer.  I also have no strong feelings about this if you wanted to make a PR to clean it up though.